internal/gcsx/file_cache_reader_test.go

```diff
@@ -208,12 +235,11 @@ func (t *fileCacheReaderTest) Test_ReadAt_RandomReadNotStartWithZeroOffsetWhenCa
 
 	readerResponse, err := t.reader.ReadAt(t.ctx, buf, int64(start))
 
-	assert.True(t.T(), errors.Is(err, FallbackToAnotherReader))
+	assert.True(t.T(), errors.Is(err, FallbackToAnotherReader), "expected %v error got %v", FallbackToAnotherReader, err)
 	assert.Zero(t.T(), readerResponse.Size)
 	job := t.jobManager.GetJob(t.object.Name, t.mockBucket.Name())
 	assert.True(t.T(), job == nil || job.GetStatus().Name == downloader.Downloading)
 	assert.NotNil(t.T(), t.reader.fileCacheHandle)
-	t.mockBucket.AssertExpectations(t.T())
```

Comment:
For reviewer: Removed this because test was completing before the NreRangeReader method got hit. Previously it was working because we had MayBe() in mock.